process.go

```diff
@@ -57,11 +57,11 @@ func Auto(ctx context.Context, owner string, repo string, currentVersion string,
 	if errors.Is(err, ErrNoNewVersion) {
 		return
 	} else if err != nil {
-		fmt.Fprintln(os.Stderr, err.Error())
+		fmt.Fprintln(os.Stderr, fmt.Sprintf("failed to check for new version: %s", err.Error()))
 		return
 	}
 
-	fmt.Fprintf(os.Stderr, "downloading new version (%s)...", newVersion)
+	fmt.Fprintf(os.Stdout, "downloading new version (%s)...", newVersion)
```

Comment:
One of the reasons I pipe the message to Stderr is to ensure that once the process is done, the terminal is automatically cleared. That was a technique being used by many.